deploy/s3/public-secret.yaml

```diff
@@ -4,13 +4,11 @@ metadata:
   name: assisted-installer-public-s3
   namespace: REPLACE_NAMESPACE
   labels:
-    app: scality
+    app: object-store
 type: Opaque
 stringData:
   aws_access_key_id: "accessKey1"
   aws_region: "us-east-1"
   aws_secret_access_key: "verySecretKey1"
   bucket: "pub-test"
-  endpoint: "http://cloudserver-front:8000"
-  s3_data_path: /mnt/data
-  s3_metadata_path: /mnt/data
+  endpoint: "http://object-store:9000"
```

Comment:
With scality we used `http://cloudserver-front:8000` because this is the default endpoint for it.
The configuration for this endpoing defined in a directory that requires root access, we don't have that when running in PSI for example (QE used or still using that).
I'm raising it just to be sure that run everything in as users and not as root.

Reply:
minio doesn't seem to care what hostname you use, but it does expose port 9000 by default.
We could create the service so that it exposed the same host and port, but I'm not sure why that would be necessary.

> I'm raising it just to be sure that run everything in as users and not as root.

This is a good point. Let me test this out in openshift as well as minikube.

Reply:
openshift as a none root user.